pkg/azurefile/controllerserver.go

```diff
@@ -222,12 +222,37 @@ func (d *Driver) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshotRequ
 		return nil, status.Error(codes.InvalidArgument, "CreateSnapshot Source Volume ID must be provided")
 	}
 
+	exists, item, err := d.snapshotExists(ctx, sourceVolumeID, snapshotName)
+	if err != nil {
+		if strings.Contains(err.Error(), "but the SourceVolumeId is different") {
+			return nil, status.Errorf(codes.AlreadyExists, "%v", err)
+		}
+		return nil, status.Errorf(codes.Internal, "failed to check if snapshot(%v) exists: %v", snapshotName, err)
+	}
+	if exists {
+		klog.V(2).Infof("snapshot(%s) exists", snapshotName)
```

Comment:
is this logging necessary?

Reply:
yes, here I think it is necessary. It can help us to debug more quickly.